schemas/oic.wk.p-schema.json

```diff
@@ -30,11 +30,9 @@
           "description": "ReadOnly, Model number as designated by manufacturer"
         },
         "mndt": {
-          "type": "string",
+          "$ref": "oic.types-schema.json#/definitions/date",
           "readOnly": true,
-          "description": "ReadOnly, Manufacturing Date as defined in ISO 8601, where the format is [yyyy]-[mm]-[dd].",
-          "pattern": "^([0-9]{4})-(1[0-2]|0[1-9])-(3[0-1]|2[0-9]|1[0-9]|0[1-9])$",
-          "format": "date-time"
+          "description": "ReadOnly, Manufacturing Date."
```

Comment:
It should and is not added (it is left untouched here). Removing it (consistently over all schema's) though is part of another pull request https://github.com/openconnectivityfoundation/core/pull/14.
But you are correct in that care will have to be taken that this patch doesn't conflict and overwrite the other patch ( by reintroduces the readonly description).